code/modules/vehicles/scooter.dm

```diff
@@ -12,38 +12,34 @@
 	AddElement(/datum/element/ridable, /datum/component/riding/vehicle/scooter)
 
 /obj/vehicle/ridden/scooter/wrench_act(mob/living/user, obj/item/I)
-	..()
+	. = ..()
 	to_chat(user, "<span class='notice'>You begin to remove the handlebars...</span>")
-	if(I.use_tool(src, user, 40, volume=50))
-		var/obj/vehicle/ridden/scooter/skateboard/improvised/S = new(drop_location())
-		new /obj/item/stack/rods(drop_location(), 2)
-		to_chat(user, "<span class='notice'>You remove the handlebars from [src].</span>")
-		if(has_buckled_mobs())
-			var/mob/living/carbon/H = buckled_mobs[1]
-			unbuckle_mob(H)
-			S.buckle_mob(H)
-		qdel(src)
+	if(!I.use_tool(src, user, 40, volume=50))
+		return TRUE
+	var/obj/vehicle/ridden/scooter/skateboard/improvised/skater = new(drop_location())
+	new /obj/item/stack/rods(drop_location(), 2)
+	to_chat(user, "<span class='notice'>You remove the handlebars from [src].</span>")
+	if(has_buckled_mobs())
+		var/mob/living/carbon/carbons = buckled_mobs[1]
+		unbuckle_mob(carbons)
+		skater.buckle_mob(carbons)
+	qdel(src)
 	return TRUE
 
 /obj/vehicle/ridden/scooter/Moved()
 	. = ..()
-	for(var/m in buckled_mobs)
-		var/mob/living/buckled_mob = m
+	for(var/mob/living/buckled_mob as anything in buckled_mobs)
 		if(buckled_mob.num_legs > 0)
 			buckled_mob.pixel_y = 5
 		else
 			buckled_mob.pixel_y = -4
 
-/obj/vehicle/ridden/scooter/buckle_mob(mob/living/M, force = FALSE, check_loc = TRUE)
-	if(!istype(M))
-		return FALSE
-	return ..()
```

Comment:
do you know why this was here in the first place?

Reply:
No idea but it's kinda useless

Reply:
Mood